<launch>

  <rosparam command="load" file="$(find dsvo_dso)/tests/cams/euroc/T_stereo.yaml" ns="dsvo"/>

  <node name="dsvo" pkg="dsvo_dso" type="dsvo_node" output="screen">
    <param name="cam0_topic" value="/cam0/image_raw" />
    <param name="cam1_topic" value="/cam1/image_raw" />
    <param name="calib0" value="$(find dsvo_dso)/tests/cams/euroc/camera0.txt" />
    <param name="calib1" value="$(find dsvo_dso)/tests/cams/euroc/camera1.txt" />
    <param name="mode" value="1" />
    <param name="preset" value="0" />
    <param name="scale_accept_th" value="10" />
  </node>

  </launch>
